gtest_parallel.py

```diff
@@ -454,7 +454,7 @@ def find_tests(binaries, additional_args, options, times):
 
       test_count += 1
 
-  return tasks
+  return sorted(tasks)
```

Comment:
Sorry, for interfering but as far as I understand if `self.last_execution_time is None` then it should correspond to bigger value in a key. As for me the following is easier to understand:

```
def __sorting_key(self):
    return (sys.float_info.max if x.last_execution_time is None else x.last_execution_time,
                ...all the other fields here is some sane order...)
```

This makes sense as we want to run first those new tests and tests with long execution time.